### ISSUE TITLE
cssStickyHeader does not work with ajaxPager

### ISSUE THREAD

**Issue by azjol** [Mottie/tablesorter#1635]
Hello,
if I use cssStickyHeaders and fill `<tbody>` element without ajaxPager, then header is sticky and everything works fine, but if I keep `<tbody>` element empty and user ajaxPager, then cssStickyHeader stop work. Is there any option, how to init cssStickyHeader every ajax request?

**Comment by Mottie** [comment]
Hi @azjol!

StickyHeaders only become sticky while scrolling over `tbody` content, if there isn't any content why would you want it to become sticky?

If you mean that the widget isn't activating after adding new content, then make sure to trigger an "update" *after* the ajax request has completed ([example](https://mottie.github.io/tablesorter/docs/example-ajax.html)).

**Comment by azjol** [comment]
Thank you very much for quick response. My code looks like this:

```
      ajaxObject: {
        type: 'GET', // default setting
        dataType: 'json',
        // Loading spinner initialize
        beforeSend: function(){ 
            $('.loading-spinner').show(); 
        },
        success: function(){ 
            $('.loading-spinner').hide(); 
            $("table").trigger("update"); 
        },
      },
```
but widget still won't start after adding content..

**Comment by Mottie** [comment]
Hmm, it's working for me - https://jsfiddle.net/Mottie/7mz8o4wv/

**Comment by Mottie** [comment]
Oh wait, I was assuming that the content being added inside the `success` ajax callback. Are you using the pager, or is that some manual implementation?

**Comment by azjol** [comment]
I am using pager

**Comment by azjol** [comment]
Is there a way to init only stickyHeader widget on ajax success?

**Comment by Mottie** [comment]
Please see https://mottie.github.io/tablesorter/docs/example-apply-widget.html

**Comment by azjol** [comment]
Nope, unfortunately it does not work. I do not know, how to update cssStickyHeaders on ajax pager data load.

**Comment by Mottie** [comment]
Please update the demo I shared to show this problem. Click on "Async requests" in the left panel to see how to emulate an ajax load. I'd set it up for you, but I have to get to work!

**Comment by azjol** [comment]
There it is https://jsfiddle.net/km6x2rdj/

**Comment by azjol** [comment]
Is it ok?

**Comment by Mottie** [comment]
Sorry, I haven't had time to look. 😿

**Comment by azjol** [comment]
It is okay @Mottie...Take a look at it when u will have a time. I think everything is set up by default, copied from documentation.

**Comment by azjol** [comment]
Hello @Mottie, Could you please look at this problem?

**Comment by Mottie** [comment]
Sorry again! I'll try to find some time this weekend.
